Makefile

```diff
@@ -20,8 +20,8 @@ dev: ## Starts the development server
 lint: ## run linter
 	npm run lint	
 
-.PHONY: install init
-install init: ## Installs dependencies
+.PHONY: install-init
+install-init: ## Installs dependencies
```

Comment:
What was the reason for that change?
targets separated by space are interchangeable. Meaning that `make init ` and `make install ` were equivalent. Quite handy because I tend to mix and match those two commands.

Reply:
ah ok, good to know, the problem is that it is not shown in help then.

Reply:
I split it